app/controllers/prx_auth/rails/sessions_controller.rb

```diff
@@ -10,19 +10,29 @@ class SessionsController < ApplicationController
     before_action :set_after_sign_in_path
 
     ID_NONCE_SESSION_KEY = 'id_prx_openid_nonce'
+    DEFAULT_SCOPES = 'openid apps'
 
     def new
       config = PrxAuth::Rails.configuration
 
+      scope =
+        if config.prx_scope.present?
+          "#{DEFAULT_SCOPES} #{config.prx_scope}"
```

Comment:
If we request something like `openid apps feeder:*` .... we get a smaller JWT to store.  Only works if we're not using this JWT to interact with any other apps.  (Augury does, a bit).